### PR TITLE
add user-provided rsa key-pair and mount rsa key-pair in InitContainers since we want to clone private git repository before run training task

### PR DESCRIPTION
**Pull request by python279**
we want to clone private git repository before run training task:

1. add user-provided rsa key-pair in vcjob spec
2. mount rsa key-pair in InitContainers

### CODE REVIEW

**Comment by wpeng102** [comment]
please make CI happy

File: pkg/controllers/job/plugins/ssh/ssh_test.go [hunk 1]
```diff
@@ -29,15 +29,27 @@ func TestSSHPlugin(t *testing.T) {
...
 			name:           "no params specified",
 			sshKeyFilePath: SSHAbsolutePath,
+			sshPrivateKey: "",
+			sshPublicKey: "",
 		},
 		{
 			name:           "--ssh-key-file-path=/a/b",
 			params:         []string{"--ssh-key-file-path=/a/b"},
 			sshKeyFilePath: "/a/b",
+			sshPrivateKey: "",
```
Comment (Thor-wl) [inline pkg/controllers/job/plugins/ssh/ssh_test.go:45]: Please execute `go fmt [files]` to make CI happy

Comment (python279) [reply pkg/controllers/job/plugins/ssh/ssh_test.go]: fixed, thx.

**Comment by Thor-wl** [comment]
CI has been fixed and please pull and rebase from master branch to make CI happy.
You can execute `git pull --rebase origin/master` to do that. @python279

**Comment by jasonliu747** [comment]
> CI has been fixed and please pull and rebase from master branch to make CI happy.
> You can execute `git pull --rebase origin/master` to do that. @python279

Hi @python279 , please follow the instruction, otherwise we can't merge this awesome work.

**Comment by python279** [comment]
> > CI has been fixed and please pull and rebase from master branch to make CI happy.
> > You can execute `git pull --rebase origin/master` to do that. @python279
> 
> Hi @python279 , please follow the instruction, otherwise we can't merge this awesome work.

Done, thx

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**